pvsreact/src/Components/Header.js

```diff
@@ -0,0 +1,46 @@
+import { useEffect, useState } from "react";
+import styles from "../CSS/HomeStyle.module.css"
+import logo from "../images/logo.png"
+import { FaUser } from "react-icons/fa";
+import axios from "axios";
+export default function Header({ token, decode }) {
+    const [name, setName] = useState("");
+    useEffect(() => {
+        fetch(`http://localhost:8080/api/getUserByEmail`, {
+            method: 'POST',
```

Comment:
shouldn't it be a GET method with a content body?

Reply:
Yes but in the controller in the back it need a post request.